doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Based on the ACRN open source project and getting started guide. This documentation is to guide people how to boot Ubuntu system as the 
+Service OS with the ACRN hypervisor. By default, in the getting started guide documentation of the ACRN open source project, Clear Linux
+is being used as the Service OS and User OS on top of the ACRN hypervisor. 
+Here we are trying to boot Ubuntu system as the Service OS and launch Clear Linux as User OS with the ACRN hypervisor on official 
+supported NUC, playing steps as below:
+
+Step #1,  Installing a Ubuntu system on the NUC with standard installing steps. 
+For example with Ubuntu 16.04.4 LTS distribution.
+Download it here: https://www.ubuntu.com/download/desktop
+Burn Ubuntu 16.04.4 image to a USB key
+Install Ubuntu system with the USB key 
+After complete installation and rebooting, you could boot Ubuntu from grub menu.
+
+Step #2,  setup proxy with you prefered ways, command line tool or system settings tool.  If you don’t need to setup proxy, you could 
+skip this step
+
+Step #3,  installing ssh service with below commands and allow “root” login over SSH optional, 
+People could just ssh into the Service OS and use 'sudo' once logged in (which would be preferable from a security standpoint), then 
+restart SSH service. Of cause, this step is not must 
+if you know how to play, just skip this step.
+# sudo apt-get install openssh-server
+# sudo service ssh status
+In /etc/ssh/sshd_config 
+Change PermitRootLogin from prohibit-password to yes
+# sudo service ssh restart
+
+Step #4, additional packages needed to install, please check the related sections of getting started guide in the ACRN open source
+project, if you don’t have an chance to setup additional packages flowing up that guid to build source. You can reference below commands
+to install directly here. 
+# sudo apt-get update 
+# sudo apt-get install git
+# sudo apt-get install libpciaccess-dev
+# sudo apt-get install gnu-efi
+# sudo apt-get install uuid-dev
+# sudo apt-get install libssl-dev
+# sudo apt-get install vim
+# sudo apt-get install libsystemd-dev
+# sudo apt-get install libevent-dev
+# sudo apt-get install libxml2-dev
+
+Step #5, Checkout source code from open source project repositories
+# git clone https://github.com/projectacrn/acrn-hypervisor
+Note: if you encounter issues with “git clone”, you can download ZIP package directly. It’s so easy.
+
+Step # 6, building the ACRN hypervisor and device model source code on Ubuntu system
+Building device model and hypervisor from source trees on NUC you just setup. You can reference the section of Getting started guide 
+of the ACRN open source project, or following up below commands to build directly here. 
+# cd acrn-hypervisor
+# make PLATFORM=uefi
+The build results are found in the build directory.
+For example:
+# ls acrn-hypervisor/build
+devicemodel
+hypervisor
+tools
+
+Steps #7,  Preparing Service OS kernel 
+You can download latest Service OS kernel from below links:
```

Comment:
yes, you may feel confusing, but for me, it's not, 
we may need to explain more here like you mentioned, 
we are Using Ubuntu system as the Service OS, but we need to customize SOS kernel to adapt hypervisor,   2 options. 1) build Ubuntu kernel with source and hypervisor patches,  i don't recommend this way for community new user.  so i  didn't mention this.
2) replace Ubuntu kernel with Clear based Service OS kernel.  this is so easy to replace,  we are using this way

Reply:
No disagreement that the steps and methodology followed is the correct one. We just need to explain some more what we're doing here (and why) because people not familiar with ACRN will be confused about this.